docs/administering/sample-config/nginx-example.conf

```diff
@@ -24,7 +24,7 @@ server {
 # Configuration for Sandstorm shell and apps, over HTTPS.
 server {
   listen 443;
-  server_name example.com *.example.com;
+  server_name example.com *.example.com ~^sandstorm-[^.]*\.example\.com$;
```

Comment:
In fact my configuration was with sandstorm installed on a subdomain.  
`server_name sandstorm.example.com ~^sandstorm-[^.]*\.example\.com$ *.sandstorm.example.com;
`  
In this case, it was needed for ghost blog static publishing:  
`sandstorm-www.host.example.com IN TXT xql6hui0fqaguiazpnk3`  

This was a while ago, maybe it is not necessary anymore?